README.md

```diff
@@ -47,41 +47,15 @@ environment for these tests, then you should activate it, otherwise you should
 create a new one. For more information on working with virtual environments see
 our [summary][virtualenv].
 
-### Install dependencies
-Install the Python packages that are needed to run our tests using pip. In a
-terminal, from the the project root, issue the following command:
-
-```bash
-$ pip install -Ur requirements.txt
-```
-
 ### Run the tests
-Tests are run using the command line. Below are a couple of examples of running
-the tests:
-
-To run all of the desktop tests against the default environment:
-
-```bash
-$ py.test --driver Firefox
-```
-
-To run against a different environment, pass in a value for `--base-url`, like
-so:
-
-```bash
-$ py.test --base-url https://treeherder.mozilla.org --driver Firefox
-```
-
-The pytest plugin that we use for running tests has a number of advanced
-command line options available. To see the options available, run
-`py.test --help`. The full documentation for the plugin can be found
-[here][pytest-selenium].
+* [Install Tox][tox]
+* Run `tox`
```

Comment:
We should keep a note on running against another base URL. Something like:

To run against a different environment, pass in a value for `--base-url`, like so:

```bash
$ tox -e tests -- --base-url https://treeherder.mozilla.org
```